spark-pipeline-engine/src/test/scala/com/acxiom/pipeline/fs/FileManagerTests.scala

```diff
@@ -171,4 +176,112 @@ class FileManagerTests extends FunSpec with Suite {
       sparkSession.stop()
     }
   }
+
+  describe("FileManager - SFTP") {
```

Comment:
Add a copy test to ensure that data can be read/written using the streams.

Reply:
I've added the test.